index.html

```diff
@@ -280,5 +280,19 @@ <h5>Skills</h5>
 				</div>
 			</div>
 		</section>
+		<section id="contact_form">
+			<div class="contact_detail">
+				<h2 class="head">Contact me</h2>
```

Comment:
- [ ] Kindly it will be better 😃  if you fix this text with a background image as you see here in your design this text is consistent with the background image, so please try to fix that to make it nice, so I suggest putting the background image at the correct position to let the text and the background image like the Figma Design💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/sheezu900/My_Potfolio/assets/83360636/7aaec593-c9cc-4e4f-a9f9-1e2c9a70ef47)  |     ![image](https://github.com/sheezu900/My_Potfolio/assets/83360636/fe89fa40-0a44-4959-818c-d6f6fca2c48d)  |